source/opt/module.h

```diff
@@ -94,6 +94,7 @@ class Module {
   // Returns a vector of pointers to type-declaration instructions in this
   // module.
   std::vector<Instruction*> types();
+  std::vector<const Instruction*> types() const;
```

Comment:
Needs a blurb. Also, it does non trivial work,  so it needs a captialized first letter.